### PR TITLE
25 v1 update front end dashboard

### PR DESCRIPTION
**Pull request by 0xMazout**

### CODE REVIEW

File: front/src/App.tsx [hunk 1]
```diff
@@ -67,11 +72,37 @@ export default function App() {
...
-                console.log("setnodeMonitor data", state.data)
                 setNodeMonitor(state.data);
             })
             .catch((error) => {
-                setNodeMonitor({status: "Down", metrics: {CPU: 0, RAM: 0, Disk: 0}});
+                setNodeMonitor({
+                    metrics: {
+                        CPU: 0.610279,
+                        RAM: 8.23754,
+                        Disk: 18
```
Comment (peterjah) [inline front/src/App.tsx:82]: do we really want displaying fake data?

File: front/src/App.tsx [hunk 2]
```diff
@@ -67,11 +72,37 @@ export default function App() {
...
+                    },
+                    status: "Up",
```
Comment (peterjah) [inline front/src/App.tsx:84]: should be Down , isn't it?

File: front/src/components/NodeActions.tsx [hunk 3]
```diff
@@ -70,39 +66,50 @@ const NodeActions: React.FC<Props> = (props: Props) => {
...
-                const link = document.createElement('a');
-                link.href = url;
-                link.setAttribute('download', 'wallet_backup.zip');
-                document.body.appendChild(link);
-                link.click();
-            })
-            .catch((error) => {
-                console.error('Error downloading wallet_backup.zip:', error);
-            })
+        if (canStop()) {
```
Comment (peterjah) [inline front/src/components/NodeActions.tsx:74]: Backup the wallet is possible even if the node is not running, as long as the host is responding. We should modify this check. Maybe some changes in the API is needed to detect if host is up even if the node is down

File: front/src/components/NodeActions.tsx [hunk 4]
```diff
@@ -70,39 +66,50 @@ const NodeActions: React.FC<Props> = (props: Props) => {
...
+                    );
+                });
+        }
     };
 
     return (
         <Grid
             container
-            spacing={2}
+            // spacing={2}
```
Comment (peterjah) [inline front/src/components/NodeActions.tsx:97]: comments to remove !

File: front/src/components/cards/ConsensusStatsCard.tsx [hunk 5]
```diff
@@ -14,50 +23,95 @@ const ConsensusStatsCard: React.FC<Props> = (props: Props) => {
...
+                                            ?.out_connection_count ?? (
+                                            <Skeleton />
+                                        )}
+                                    </Typography>
+                                    <Typography variant="body2">
+                                        Outgoing
+                                    </Typography>
+                                </Grid>
+                            </Grid>
+                        </Grid> */}
```
Comment (peterjah) [inline front/src/components/cards/ConsensusStatsCard.tsx:114]: to remove ?

File: front/src/components/cards/NetworkStatsCard.tsx [hunk 6]
```diff
@@ -18,7 +26,68 @@ const NetworkStatsCard: React.FC<Props> = (props: Props) => {
...
+                                    <Typography variant="body2">
+                                        Outgoing
+                                    </Typography>
+                                </Grid>
+                            </Grid>
+                        </Grid>
+                    </Grid>
+                </CardContent>
+            </Card>
+            {/* <Card
```
Comment (peterjah) [inline front/src/components/cards/NetworkStatsCard.tsx:88]: commented code

File: front/src/components/NodeActions.tsx (hunk 3 above)
Comment (0xMazout) [reply front/src/components/NodeActions.tsx:74]: I think we don't handle the case if the host is not connected on any point of the app , need to verify this but in my memories we don't handle this case

**Review by peterjah — APPROVED** [review]

File: front/src/components/cards/NodeStakingInfoCard.tsx [hunk 7]
```diff
@@ -0,0 +1,174 @@
...
+                                    <Typography variant="subtitle2" width="30%" marginLeft={3}>
+                                        Wallet Balance:
+                                    </Typography>
+                                    
+                                    <Typography
+                                        variant="h6"
+                                        marginLeft={5}
+                                    >
+                                        {props.nodeMonitor?.wallet_infos
+                                            ?.Final_rolls ?? (
```
Comment (Thykof) [inline front/src/components/cards/NodeStakingInfoCard.tsx:77]: I think you are showing final rolls where the title is about wallet balance and the following tooltip is about thread. same for 2 block bellow

File: front/src/App.tsx [hunk 8]
```diff
@@ -11,17 +11,15 @@ import Header from "./components/Header";
 import Install from "./pages/Install";
 import Manager from "./pages/Manager";
 
-import { localApiGet, request } from "./request";
+import { apiPost, localApiGet, nodeApiPost, request } from "./request";
 
-const getStatusFromNodeApi = async (host: string): Promise<any> => {
-    const res = await request("POST", `http://${host}:33035/api/v2`, {
+const getStatusFromNodeApi = (host: string): Promise<any> => {
+    return  nodeApiPost(`http://${host}:33035/api/v2`, {
```
[the remark this answers is not in the document]
Comment (0xMazout) [reply front/src/App.tsx:17]: That's right eslint is not installed in this project

**Review by Thykof — APPROVED** [review]

**Comment by Thykof** [comment]
Ci is read because I canceled workflow to save quotas

File: front/src/App.tsx [hunk 9]
```diff
@@ -32,6 +30,10 @@ const getNodes = async (): Promise<any> => {
     return localApiGet("nodes");
 };
 
+const getLogs = async (id: string): Promise<any> => {
```
[the remark this answers is not in the document]
Comment (0xMazout) [reply front/src/App.tsx:33]: https://github.com/massalabs/thyra-node-manager-plugin/issues/32

**Pull request merged**